lectures/scipy.md

```diff
@@ -442,7 +443,7 @@ We leave you to investigate the [set of available routines](http://docs.scipy.or
 (sp_ex1)=
 ### Exercise 1
 
-Previously we discussed the concept of {ref}`recursive function calls <recursive_functions>`.
+In Chapter 16, we will discuss the concept of {ref}`recursive function calls <recursive_functions>`.
```

Comment:
@Horace510 , we need to avoid using numbers like 16 here because they can easily become wrong if the document changes.  You need to use a reference.  See here: https://jupyterbook.org/reference/cheatsheet.html#reference-documents

Reply:
@Horace510 are you able to update this?

Reply:
I have made an update to resolve this